packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -193,10 +194,10 @@ const prompts = () => {
     const validSiteId = (s) =>
         /^[a-z0-9_-]+$/i.test(s) || 'Valid characters are alphanumeric, hyphen, or underscore'
 
-    // To see definitions for Commerce API configuration values, refer to these
-    // doc --> https://developer.salesforce.com/docs/commerce/commerce-api/guide/commerce-api-configuration-values.html.
+    // To see definitions for Commerce API configuration values, go to
+    // https://developer.salesforce.com/docs/commerce/commerce-api/guide/commerce-api-configuration-values.
     const defaultCommerceAPIError =
-        'Invalid format. Use docs to find more information about valid configurations: https://developer.salesforce.com/docs/commerce/commerce-api/guide/commerce-api-configuration-values.html'
+        'Invalid format. Use docs to find more information about valid configurations: https://developer.salesforce.com/docs/commerce/commerce-api/guide/commerce-api-configuration-values'
```

Comment:
Thank you for fixing the links

Reply:
BTW, the link still works with the `.html` but removing it will give us some future proofing in case the platform changes the way it handles URLs